pre-commit/app_tests/utils/app_parser.py

```diff
@@ -106,15 +93,21 @@ def find_app_json_name(self, json_filenames):
         return filtered_json_filenames[0]
 
     @property
-    def _app_json_filepath(self):
-        return os.path.join(self._app_code_dir, self.app_json_name)
+    def _app_json_filepath(self) -> Path:
+        return Path(self._app_code_dir) / self.app_json_name
 
     @cached_property
     def app_json(self):
         # Gets the loaded json, preserving key order
-        with open(self._app_json_filepath) as f:
-            json_content = json.loads(f.read(), object_pairs_hook=OrderedDict)
-        return json_content
+        try:
+            json_content = json.loads(
+                self._app_json_filepath.read_text(), object_pairs_hook=OrderedDict
+            )
```

Comment:
Dict ordering in python has actually been guaranteed since 3.6, I don't think we need this anymore?
```suggestion
            json_content = json.loads(self._app_json_filepath.read_text())
```

Reply:
Yep removing